src/test/rgw/bucket_notification/test_bn.py

```diff
@@ -1432,10 +1403,9 @@ def ps_s3_creation_triggers_on_master(external_endpoint_address=None, ca_locatio
         key.delete()
     # delete the bucket
     conn.delete_bucket(bucket_name)
-    if proc:
-        clean_rabbitmq(proc)
 
 
+@attr('AMQP test')
```

Comment:
please also add attribute to: 
`test_ps_s3_creation_triggers_on_master_external()`
`test_ps_s3_creation_triggers_on_master_ssl()`